src/raw.js

```diff
@@ -30,9 +30,16 @@ assign(Raw.prototype, {
     return this
   },
 
-  timeout(ms) {
+  timeout(ms, {cancel} = {}) {
     if(isNumber(ms) && ms > 0) {
       this._timeout = ms;
+      if (cancel) {
```

Comment:
Unnecessary nested if-statements here. `if (cancel && this.client.canCancelQuery)`